src/blockbroadcast.jl

```diff
@@ -30,13 +30,16 @@ BroadcastStyle(::PseudoBlockStyle{M}, ::BlockStyle{N}) where {M,N} = BlockStyle(
 
 # sortedunion can assume inputs are already sorted so this could be improved
 sortedunion(a,b) = sort!(union(a,b))
+sortedunion_tuple(a::Tuple, b::Tuple) = (a..., b...)
```

Comment:
Isn't this not really a union since it doesn't remove elements that `a` and `b` might have in common?

Could call it `disjointsortedunion_tuple` or something like that to clarify the inputs should also be disjoint.

Reply:
Agreed. Also, I'm not sure how I can fix the issues that come from this change..

Reply:
I think the only "solution" might be to somehow avoid this codepath, which I've not had enough time to try